pkg/service/aws/session.go

```diff
@@ -48,6 +49,7 @@ func getSessionUsingAwsProfile(profile string) (*session.Session, error) {
 
 	sess, err := session.NewSessionWithOptions(
 		session.Options{
+			Config:  aws.Config{Region: aws.String("us-west-2")},
```

Comment:
Can we get it from config?

Reply:
out of the scope of this PR, another PR addresses this

Reply:
merged changes of that PR.